packages/drivers/routerlicious-driver/src/documentDeltaConnection.ts

```diff
@@ -47,6 +48,7 @@ export class R11sDocumentDeltaConnection extends DocumentDeltaConnection
             tenantId,
             token,  // Token is going to indicate tenant level information, etc...
             versions: protocolVersions,
+            relayUserAgent: [client.details.environment, ` driverVersion:${driverVersion}`].join(";"),
```

Comment:
In Alfred's index.ts, we only use the `driverVersion` after parsing the `relayUserAgent` info. What do we use `client.details.environment` for?

Reply:
Nothing yet, but I figured we might have use for the information at a later time

Reply:
Is there a chance the extra parsing can cause added latency to the websocket requests (especially given high volume situations)? If not, then I agree it probably does not hurt to include `client.details.environment`. But if there is indeed a small performance penalty due to parsing, do you think it would make sense to not include that info, since we don't have any use for it now?

Reply:
I think that's a fair point. I tried it out locally with 2-8 key-value pairs and it ranged from 0.04-0.06ms. Do we think that's unacceptable for performance?

Reply:
Probably not :) I think it's probably negligible. I was under the impression that it would be worse. Thanks a lot for running the experiment! Please proceed and feel free to ignore this comment 😄